docs/plugin_developers_guide/01_Fledge_plugins.rst

```diff
@@ -41,11 +41,109 @@ Plugins
+| Entry                  | Required by            | Description          | Fields                                                |
++========================+========================+======================+=======================================================+
+| plugin                 | all                    | Basic plugin name    | description: "<Describe plugin>" |br|                 |
+|                        |                        | and information      | type: "string" |br|                                   |
+|                        |                        |                      | name: "<plugin name>" |br|                            |
+|                        |                        |                      | readonly: "true"                                      |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| enable                 | Filter |br|            | Enable/disable       | description: "<Describe plugin being enabled>" |br|   |
+|                        | Notification |br|      | plugin function      | type: "boolean"                                |br|   |
+|                        | Notification delivery  |                      | default: "false"                               |br|   |
+|                        |                        |                      | displayName: "<name for UI display>"           |br|   |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| asset                  | South |br|             | Asset where data     | description: "<name of asset being monitored>" |br|   |
+|                        | Notification           | fields are deposited | type: "string"                                 |br|   |
+|                        |                        |                      | default: ""                                    |br|   |
+|                        |                        |                      | displayName: "<name for UI display>"           |br|   |
+|                        |                        |                      | mandatory: "true"                              |br|   |
+|                        |                        |                      | order: "<order of display in UI>"                     |
++------------------------+------------------------+----------------------+-------------------------------------------------------+
+| source                 | North                  | Choice of data to    | description: "<Resource being forwarded>" |br|        |
+|                        |                        | forward from Foglamp | type: "enumeration"                  |br|             |
```

Comment:
Use of Foglamp name in Fledge documentation. Note also the table formatting issue here with white labelling.